modules/ROOT/pages/available-menu-items.adoc

```diff
@@ -156,6 +156,12 @@ include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 :!pluginpage:
 
+:plugincategory: premium
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginpage: autocorrect.adoc
```

Comment:
This should only be set when the page is different to the plugincode. Also if using it, then it needs to be unset after the include.
```suggestion
```